doc/app-flows.md

```diff
@@ -0,0 +1,33 @@
+# App Flows
+
+## Main App Flow
+
+1. User receives a text with a redemption link.
+
+2. Check if the customer is logged in with `Get Current Customer`.
+
+3. Use `Fetch Unclaimed Orders` to get information about the order.
+
+4. Check in your system if a customer API token exists for that customer.
+     - If the token does not exist, follow [New Customers](#new-customers).
+     - If the token does exist, follow [Existing Customers](#existing-customers).
+
+5. Claim the order with the redemption code using `Claim Orders`.
+
+6. Provide failsafe options for order events. At a minimum, have a prompt for `arrived`. Use `Update Orders` to provide these event updates to FlyBuy. This section also lists the possible state changes.
+
+7. Order history can be shown using `Fetch Claimed Orders`.
```

Comment:
Maybe this goes in a section about fetching orders along with fetching open orders?